src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalysisResult.cs

```diff
@@ -13,195 +14,148 @@ namespace Microsoft.CodeAnalysis.Diagnostics
     /// 1. Local and non-local diagnostics, per-analyzer.
     /// 2. Analyzer execution times, if requested.
     /// </summary>
-    internal partial class AnalysisResult
```

Comment:
This type has been renamed to AnalysisResultBuilder and a new public type AnalysisResult has been added - unfortunately the diff here is not helpful to review.